### PR TITLE
embeds & message refactor and tweaks

### LINKED ISSUE #101: Move all strings to `strings.rs`
## Rationale

`strings.rs` is a really good starting point for future translations, so we should move as many strings to there.

### PR DESCRIPTION
**Pull request by afonsojramos**
* Changed all messages to use embeds
* Centralised message creation and edition in `utils.rs`
* Add emojis to basic commands
* Fix misuse of `QUEUE_IS_EMPTY` vs `NOTHING_IS_PLAYING`
* Closes #101

### CODE REVIEW

**Review by joao-conde — APPROVED** [review]

File: src/commands/play.rs [hunk 1]
```diff
@@ -75,40 +79,28 @@ pub async fn _play(
...
+                    edit_embed_response(&ctx.http, interaction, embed).await?;
                 }
             },
             EnqueueType::PLAYLIST => {
-                interaction
-                    .edit_original_interaction_response(&ctx.http, |response| {
-                        response.content("Added playlist to queue!")
-                    })
-                    .await?;
+                edit_response(&ctx.http, interaction, "⌛  Added {} tracks to queue").await?;
```
Comment (aquelemiguel) [inline src/commands/play.rs:96]: This is busted! It isn't editing the existing embed and probably missing a `format!`. 

image

Reconsider ⌛ here as well.

File: src/commands/play.rs [hunk 2]
```diff
@@ -75,40 +79,28 @@ pub async fn _play(
...
-                            r.content(" ").add_embed(embed)
-                        })
-                        .await?;
+                    edit_embed_response(&ctx.http, interaction, embed).await?;
                 }
                 PlayFlag::DEFAULT => {
                     let track = queue.last().unwrap();
-                    let embed = create_queued_embed("Added to queue", track, estimated_time).await;
+                    let embed =
+                        create_queued_embed("⌛  Added to queue", track, estimated_time).await;
```
Comment (aquelemiguel) [inline src/commands/play.rs:90]: Reconsider ⌛ here as well.

File: src/commands/play.rs [hunk 3]
```diff
@@ -75,40 +79,28 @@ pub async fn _play(
             EnqueueType::URI | EnqueueType::SEARCH => match flag {
                 PlayFlag::PLAYTOP => {
                     let track = queue.get(1).unwrap();
-                    let embed = create_queued_embed("Added to top", track, estimated_time).await;
+                    let embed =
+                        create_queued_embed("⌛  Added to top", track, estimated_time).await;
```
Comment (aquelemiguel) [inline src/commands/play.rs:83]: In retrospect, using ⌛ looks like it's still loading. Maybe a 👍 or ✅ would be more intuitive.

File: src/commands/seek.rs [hunk 4]
```diff
@@ -53,7 +53,7 @@ pub async fn seek(
     create_response(
         &ctx.http,
         interaction,
-        &format!("Seeked current track to **{}**!", seek_time),
+        &format!("⏭  Seeked current track to **{}**!", seek_time),
```
Comment (aquelemiguel) [inline src/commands/seek.rs:56]: ⏩ (meaning fast-forward) makes more sense here as opposed to ⏭️ (meaning [next track](https://emojipedia.org/emoji/%E2%8F%AD%EF%B8%8F/)).

File: src/commands/resume.rs [hunk 5]
```diff
@@ -24,11 +24,11 @@ pub async fn resume(
     let queue = handler.queue();
 
     if queue.is_empty() {
-        return create_response(&ctx.http, interaction, QUEUE_IS_EMPTY).await;
+        return create_response(&ctx.http, interaction, NOTHING_IS_PLAYING).await;
     }
 
     if queue.resume().is_ok() {
-        return create_response(&ctx.http, interaction, "Resumed!").await;
+        return create_response(&ctx.http, interaction, "▶️  Resumed!").await;
```
Comment (aquelemiguel) [inline src/commands/resume.rs:31]: Spaces are being ignored?

image

**Review by aquelemiguel — CHANGES_REQUESTED** [review]

**Review by joao-conde — APPROVED** [review]

File: src/commands/resume.rs (hunk 5 above)
Comment (afonsojramos) [reply src/commands/resume.rs:31]: Wow, this was harder to fix than it should... It is Discord's fault, but eventually after some searching I found a fix on a [reddit comment](https://www.reddit.com/r/Discord_Bots/comments/qjageg/comment/hiouo52/?utm_source=reddit&utm_medium=web2x&context=3)!

Comment (afonsojramos) [reply src/commands/resume.rs:31]: Nevermind, this is what happens on MacOS 
image

Comment (afonsojramos) [reply src/commands/resume.rs:31]: Okay, back on track! This one (https://www.compart.com/en/unicode/U+2002) works on both platforms!
However, we'll need to remember that it is a different space on new strings!
image

**Review by aquelemiguel — CHANGES_REQUESTED** [review]
Good changes, love having all messages centralized on `strings.rs`.

Still looks like the "Added playlist to queue" message isn't replacing the "Searching..." placeholder, could you please have a look at that?

image

**Comment by afonsojramos** [comment]
> Good changes, love having all messages centralized on `strings.rs`.
> 
> Still looks like the "Added playlist to queue" message isn't replacing the "Searching..." placeholder, could you please have a look at that?
> 
> image
> 

This seems like a mismatch with the previous PR that was merged, I'll look into it!

**Review by aquelemiguel — APPROVED** [review]

**Pull request merged**